docs/faq.rst

```diff
@@ -0,0 +1,114 @@
+.. _faq:
+
+Frequently asked questions
+==========================
+
+General fairness questions
+--------------------------
+
+Where can I learn more about fairness in machine learning?
+    For a high level view of the subject, please see the :ref:`further reading <further_reading>`
```

Comment:
I think that we should replace "further reading" with "further resources" and just send the users there. That page should include both general links as well as academic papers, so we don't need to call out academic papers in this answer.

Reply:
I've renamed "Further Reading" to "Further Resources"